components/stack_exchange/sources/new-answers-for-questions/new-answers-for-questions.js

```diff
@@ -0,0 +1,74 @@
+const stack_exchange = require('../../stack_exchange.app');
+
+module.exports = {
+  key: "stack_exchange-new-answers-for-questions",
+  name: "New Answers for Specific Questions",
+  description: "Emits an event when a new answer is posted in one of the specified questions",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    stack_exchange,
+    db: "$.service.db",
+    siteId: { propDefinition: [stack_exchange, "siteId"] },
+    questionIds: {
+      propDefinition: [
+        stack_exchange,
+        "questionIds",
+        c => ({ siteId: c.siteId }),
+      ],
+    },
+    timer: {
+      type: '$.interface.timer',
+      default: {
+        intervalSeconds: 60 * 15, // 15 minutes
+      },
+    },
+  },
+  hooks: {
+    async activate() {
+      const fromDate = this._getCurrentEpoch();
+      this.db.set("fromDate", fromDate);
+    },
+  },
+  methods: {
+    _getCurrentEpoch() {
+      // The StackExchange API works with Unix epochs in seconds.
+      const epochInMs = +new Date();
```

Comment:
Same small feedback from other PR

Reply:
Will do.

Reply:
Yup, will fix all the occurrences within this component.